hooks/level_up.php

```diff
@@ -16,7 +16,7 @@ function hook_level_up($container, $args = 0)
     //Check if player has leveled up
     if ($args->exp >= $args->max_exp) {
         //Update the current player variable ($args)
-        //$args->exp = $args->exp - $args->max_exp;  //Dafaq?
+        $args->exp = $args->exp - $args->max_exp;  //Figured it out, we're setting Exp back to 0 + however much above it we were.... Leveling
```

Comment:
Sounds like a comment of mine.
I don't always have my proudest moments while coding.